packages/flutter/lib/src/painting/binding.dart

```diff
@@ -79,6 +81,42 @@ mixin PaintingBinding on BindingBase, ServicesBinding {
     super.evict(asset);
     imageCache.clear();
   }
+
+  /// Listenable for OSes system fonts change event.
+  ///
+  /// System fonts can change when system installs or removes new font. To
+  /// correctly reflect the change, it is important to relayout text related
+  /// widgets when this happens.
+  Listenable systemFonts;
+
+  Set<VoidCallback> _systemFontsCallbacks;
+
+  /// System channel message handler in painting layer.
+  ///
+  /// This handler checks for system fonts change notification and invokes
+  /// callbacks in [systemFontsOCallbacks].
```

Comment:
nit: spelling within [].

Reply:
Actually, since this is an override you could just remove this doc comment altogether. It documents implementation details anyways.